app/src/main/java/com/spendster/presentation/ViewPagerAdapter.java

```diff
@@ -0,0 +1,26 @@
+package com.spendster.presentation;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import java.util.ArrayList;
+
+public class ViewPagerAdapter extends FragmentPagerAdapter {
+    private ArrayList<Fragment> slideFragments;
+
+    public ViewPagerAdapter(FragmentManager fragmentManager, ArrayList<Fragment> slideFragments) {
+        super(fragmentManager);
+        this.slideFragments = slideFragments;
+    }
+
+    @Override
+    public int getCount() {
+        return slideFragments.size();
```

Comment:
@SimonMaha add a check to be sure list is not null. We will get Nullpointer exception otherwise.
if(this.slideFragments == null) {
   returt 0;
}